ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_searching_sorting.py

```diff
@@ -337,3 +337,38 @@ def test_jax_numpy_sort_complex(
         a=x[0],
         test_values=False,
     )
+
+
+# where
+@handle_frontend_test(
+    fn_tree="jax.numpy.where",
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("numeric")),
+    test_with_out=st.just(False),
+    native_arrays=st.just(True),
+)
+def test_jax_numpy_where(
+    *,
+    dtype_and_x,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+    native_array
+):
+    input_dtype, x = dtype_and_x
+    x = x[0]
+    condition = x > 0.5
+    x1 = x * 2
+    x2 = x * -2
+
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
```

Comment:
tis should be `["bool", input_dtype, input_dtype]` because thats what we're passing here

Reply:
Thanks, I have updated it!